airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/HeartbeatController.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.container_orchestrator;
+
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Produces;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import java.util.Map;
+
+/**
+ * Creates a controller that returns a 200 JSON response on any path requested.
+ * <p>
+ * This is intended to stay up as long as the Kube worker exists so pods spun up can check if the
+ * spawning Kube worker still exists.
+ */
+// @TODO remove this annotation
```

Comment:
Is this TODO still valid?

Reply:
Depends.  How do we want to handle auth within the service itself?  Without this annotation this endpoint returns a `403`.  This endpoint doesn't need any auth as it's not exposing any information that would require an authenticated user.

Reply:
This shouldn't need authentication.